src/daily_output.ts

```diff
@@ -226,40 +226,40 @@ export class DailyOutput {
 		const node = edge.node;
 		const txId = node.id;
 		const ownerAddress = node.owner.address;
-		const tip = +node.quantity.winston;
 		const tags = node.tags;
 		const dataSize = +node.data.size;
-		const entityTypeTag = tags.find((tag) => tag.name === 'Entity-Type')?.value;
-		const bundledIn = node.bundledIn?.id;
-		const bundleVersion = tags.find((tag) => tag.name === 'Bundle-Version')?.value;
-		const bundleTipType = tags.find((tag) => tag.name === 'Tip-Type')?.value;
-		const isMetadataTransaction = !!entityTypeTag && !bundleVersion;
-		const isBundleTransaction = !!bundleVersion;
-
-		const previousTimestamp = this.latestTimestamp * 1000;
-		const previousBlockHeight = this.previousData.blockHeight;
-		const previousDate = new Date(previousTimestamp);
 
 		const height = edge.node.block.height;
-
+		const previousBlockHeight = this.previousData.blockHeight;
 		if (previousBlockHeight >= height) {
 			throw new Error('That block was already processed!');
 		}
 
 		const queryTimestamp = edge.node.block.timestamp * 1000;
 		const queryDate = new Date(queryTimestamp);
-
+		const previousTimestamp = this.latestTimestamp * 1000;
+		const previousDate = new Date(previousTimestamp);
 		if (this.isNewESTDate(previousDate, queryDate)) {
 			this.resetDay();
 		}
-
 		if (this.isNewESTWeek(previousDate, queryDate)) {
 			this.resetWeek();
 		}
 
+		const boostValue = +(tags.find((tag) => tag.name === 'Boost')?.value || '1');
+		const tip = +node.quantity.winston;
+		const fee = +node.fee.winston;
+		const unboostedFee = fee / boostValue;
+		const tipPercentage = unboostedFee / tip;
+		const entityTypeTag = tags.find((tag) => tag.name === 'Entity-Type')?.value;
+		const bundleVersion = tags.find((tag) => tag.name === 'Bundle-Version')?.value;
+		const isTipPercentageValid = !tip ? 0 : tipPercentage + Number.EPSILON >= 15;
```

Comment:
Are we adding this to cover the rounding?

Reply:
Yes! I'll add a comment on that